keyboards/gray_studio/hb85/rules.mk

```diff
@@ -31,7 +31,7 @@ F_CPU = 12000000
 BOOTLOADER = bootloadHID
 
 # build options
-BOOTMAGIC_ENABLE = lite
+BOOTMAGIC_ENABLE = no
```

Comment:
Is the `K00` (F3) while plugging in not the Bootmagic Lite method?

Reply:
K00 is actually the default bootmagic lite configuration. It was my hope that the bootmagic lite would not be the same as the default reset so as to give people different "reset" options. I mapped it to Esc, but it didn't work.